### DISCUSSION TITLE
可以回家了

### DISCUSSION THREAD

**Discussion by Arataki-Amerike** [share121/inter-knot#4804] [其他]
半夜被热起来发现发烧了，睡不着故发绳网空间，特此告知。
![Uploading IMG_20240912_042427.jpg…]()

**Comment by Arataki-Amerike** [comment]
图片怎么上传不了
![Uploading IMG_20240912_042427.jpg…]()

**Reply by Yizakl** [reply]
等‘uploading’没了就好

**Comment by March-777th** [comment]
虽然回家很爽，但还是祝早日康复<img src="https://bbs-static.miyoushe.com/static/2024/07/31/2dafa973b0fb29e5342403c4ead8af29_7443113916298582946.png" alt="三月七 比划" width="75" >
